hoomd/mpcd/pytest/test_snapshot.py

```diff
@@ -0,0 +1,261 @@
+# Copyright (c) 2009-2023 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+import hoomd
+from hoomd import Simulation
+import numpy as np
+import pytest
+
+# default testing configuration, used in setting values below
+test_positions = np.array([[1, 2, 3], [-9, -6, -3], [7, 8, 9]])
+test_velocities = np.array([[-1, -2, -3], [4, 5, 6], [-7, -8, -9]])
+test_typeids = np.array([1, 0, 2])
+test_types = ["M", "P", "H"]
+test_mass = 1.5
+
+
+@pytest.fixture
+def snap():
+    snap_ = hoomd.Snapshot()
+    snap_.configuration.box = [20, 20, 20, 0, 0, 0]
+    return snap_
+
+
+def test_make(snap):
+    if snap.communicator.rank == 0:
+        assert snap.mpcd.N == 0
+        assert len(snap.mpcd.position) == 0
+        assert len(snap.mpcd.velocity) == 0
+        assert len(snap.mpcd.typeid) == 0
+        assert len(snap.mpcd.types) == 0
+        assert snap.mpcd.mass == 1.0
+
+        # resize and test default fillings
+        snap.mpcd.N = 3
+        assert snap.mpcd.N == 3
+        assert len(snap.mpcd.position) == 3
+        assert len(snap.mpcd.velocity) == 3
+        assert len(snap.mpcd.typeid) == 3
+        np.testing.assert_array_equal(snap.mpcd.position, np.zeros((3, 3)))
+        np.testing.assert_array_equal(snap.mpcd.velocity, np.zeros((3, 3)))
+        np.testing.assert_array_equal(snap.mpcd.typeid, np.zeros(3))
+
+        # test setting values in the snapshot
+        snap.mpcd.N = test_positions.shape[0]
+        snap.mpcd.position[:] = tuple(test_positions)
+        snap.mpcd.velocity[:] = list(test_velocities)
+        snap.mpcd.typeid[:] = test_typeids
+        snap.mpcd.types = test_types
+        snap.mpcd.mass = test_mass
+        np.testing.assert_array_equal(snap.mpcd.position, test_positions)
+        np.testing.assert_array_equal(snap.mpcd.velocity, test_velocities)
+        np.testing.assert_array_equal(snap.mpcd.typeid, test_typeids)
+        np.testing.assert_array_equal(snap.mpcd.types, test_types)
+
+
+# test that snapshot can be resized without losing data
+def test_resize(snap):
```

Comment:
```suggestion
def test_resize(snap):
    """Test that snapshot can be resized without losing data."""
```
Personally, I like to put these little descriptions of what the test is doing in a one line docstring.